### PR TITLE
refactor(buttons): convert buttons in scans and credentials list views

### PR DESCRIPTION
**Pull request by jenny-s51**
## What's included

...
Converts PF3 buttons to PF4 buttons in list view of Scans and Credentials screens. 

**This includes the icons on the right of every list item, and the "Merge Reports" button on the Scans screen.**

## Notes
**The changes in this PR are aligned with the code and feedback in #136** 

## Example

...
### Before
Screen Shot 2022-08-15 at 2 07 09 PM
Screen Shot 2022-08-15 at 2 07 24 PM
Screen Shot 2022-08-15 at 2 07 50 PM
Screen Shot 2022-08-15 at 11 43 26 AM

### After
Screen Shot 2022-08-19 at 3 41 06 PM
Screen Shot 2022-08-15 at 2 03 55 PM
Screen Shot 2022-08-15 at 2 04 43 PM
Screen Shot 2022-08-15 at 2 06 07 PM
Screen Shot 2022-08-15 at 2 06 38 PM

## Updates issue/story

[DISCOVERY-149](https://issues.redhat.com/browse/DISCOVERY-149)
#134

### CODE REVIEW

File: src/components/scans/scans.js [hunk 1]
```diff
@@ -81,13 +81,14 @@ class Scans extends React.Component {
     return (
       <div className="form-group">
         <Tooltip key="mergeButtonTip" content="Merge selected scan results into a single report">
-          <ButtonPf3
+          <Button
             id="merge-reports"
-            disabled={viewOptions.selectedItems.length <= 1}
+            variant={ButtonVariant.secondary}
```
Comment (cdcabrera) [inline src/components/scans/scans.js:86]: required:
investigate with UX @bclarhk to see if this should be a primary button

**Review by cdcabrera — CHANGES_REQUESTED** [review]
Double check with Ben on the button but otherwise, lgtm!

**Review by bclarhk — CHANGES_REQUESTED** [review]
Yeah, not sure why the merge reports is/was a secondary button. It's the only real action on the page. Let's make it the primary button.

**Review by cdcabrera — APPROVED** [review]

**Review by jenny-s51 — COMMENTED** [review]
Thank you for your reviews and feedback @cdcabrera @bclarhk ! 

I've switched the button to use the `primary` variant, and have uploaded a screenshot with the updated appearance of the "merge reports" button.

**Pull request merged**